rust/trace/src/chrome_trace_dump.rs

```diff
@@ -82,6 +82,42 @@ where
     serde_json::from_reader(input).map_err(Error::Json)
 }
 
+/// Save tracing data to path pointed to by the environment variable TRACE_OUTPUT, using the Trace
+/// Viewer format. Save path defaults to `./target/trace_output.trace`. Trace file can be opened
+/// with the Chrome browser by visiting the URL `about:tracing`. If `sorted_chronologically` is true
+/// then sort output traces chronologically by time of creation.
+pub fn save_traces(sorted_chronologically: bool) {
+    use std::env;
+
+    let traces = if sorted_chronologically {
+        super::samples_cloned_sorted()
+    } else {
+        super::samples_cloned_unsorted()
+    };
+
+    let trace_output_path = match env::var("TRACE_OUTPUT") {
+        Ok(output_path) => output_path,
+        Err(_) => {
+            println!("Environment variable TRACE_OUTPUT not set, defaulting to ./target/trace_output.trace");
+            String::from("./target/trace_output.trace")
```

Comment:
I would discourage a fixed path name, which means you'll always overwrite any existing files. One compromise would be using the PID of the current process?

Reply:
@cmyr or perhaps something that includes time + date?

Reply:
That's totallly an option; the downside is that it drags in another dependency.

Reply:
@cmyr How about we let the calling application decide how they want to name their path? I have changed the function so that it now expects a path, and thus doesn't default to anything---thoughts?

Reply:
I think that is the best approach.